browser/notifications/notification_platform_bridge_brave_custom_notification.cc

```diff
@@ -0,0 +1,178 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/notifications/notification_platform_bridge_brave_custom_notification.h"
```

Comment:
why do we have a bridge when these aren't using platform notifications?

Reply:
AdsServiceImpl was built around using `NotificationPlatformBridge`, and originally Android was going to be in a separate PR. Since Android is part of this PR, we can rename it to something like `AdsNotificationViewController` or something similar. Let me know your thoughts.